src/mongoose_cleaner.erl

```diff
@@ -78,10 +80,12 @@ cleanup_modules(Node) ->
     Retries = 1,
     case global:trans(LockRequest, C, Nodes, Retries) of
         aborted ->
-            ?LOG_INFO(#{what => cleaner_trans_aborted,
-                        text => <<"mongoose_cleaner failed to get global lock">>,
-                        lock_key => LockKey}),
-            {ok, aborted};
+            ?LOG_ERROR(#{what => cleaner_trans_aborted,
+                         text => <<"mongoose_cleaner failed to get global lock">>,
+                         lock_key => LockKey}),
+            %% Wait and retry
+            timer:sleep(rand:uniform(1000)),
+            cleanup_modules(Node);
```

Comment:
should not happen, unless other node stuck performing cleaning indefinitely (so, not releasing the lock).
Just giving up does not make sense.
I could retry limited amount of times and do cleaning without any transaction, if I fail to get lock - could make a spike of load from several cleaners running at once, but otherwise should be safe (at least safer than keeping garbage in the tables).